modules/developers-guide/examples/galois/Galois.mo

```diff
@@ -113,23 +109,27 @@ module Galois {
   };
 
   public func elemFromBit(bit : Bool) : Elem {
-    if bit { { unbox = 1 } } else { { unbox = 0 } }
+    if bit {
+      { unbox = 1 }
+    } else {
+      { unbox = 0 }
+    }
```

Comment:
what about `{ unbox = if bit { 1 } else { 0 } }`

Reply:
Sure, but I don't want to change Enzo's code in two places. Not even sure all that unbox stuff is even necessary.

Reply:
This is basically copied from the enzo/motoko-qr repository... it might be bit high-fallutin for sample code anyway - I just want to make sure it at least parses and compiles as included in the doc